content/blog/posts/this-is-how-we-watchdog.md

```diff
@@ -0,0 +1,40 @@
+---
+title: This is how we watchdog
+date: 2020-12-21
+author: Eva Ruth Moravec
+---
+We frequently mention that we “watchdog” the data that we work with as much as possible, but I thought I’d peel that back a bit. What do TJI’s oversight efforts look like? 
+
+Each month, I get new data from the Texas Office of the Attorney General reflecting the previous month's reported deaths in custody and shootings of and by law enforcement officers. I add this new data to TJI's main data sets. Overnight, bots look for changes to the data sets, which are then pushed through our processing pipeline. 
+
+The data-entry process is still quite manual, and I often find errors in the submitted reports – everything from misspellings and wrong names to transposed dates of death that result in negative ages and narratives obviously pulled from a different report. I note these, as well as missing reports (ones that were not filed within the required 30-day time period). **I recently went through this process and wanted to document and narrate what this effort looks like.** 
+
+First observation: There were quite a few custodial deaths in November – 144 *new* deaths for that month alone, when we usually see around 90. Next, I noted a few missing custodial death reports: 
```

Comment:
- I am curious to know how you figure out whether there are missing reports. It might be also nice to explain that process.
- What does "new" deaths mean? Is it just the total number of deaths in Nov or *additional* deaths in Nov compared to Oct?

Reply:
Good points, thanks. 
Added language for the first point detailing that I find "missing" reports via media 
Also changed new to "144 deaths reported in that month alone"